ui/app/components/configure-ssh-secret.js

```diff
@@ -18,7 +18,8 @@ import { action } from '@ember/object';
  */
 export default class ConfigureSshSecretComponent extends Component {
   @action
-  saveConfig(data) {
-    this.args.saveConfig(data);
+  saveConfig(event) {
+    event.preventDefault();
+    this.args.saveConfig(event);
```

Comment:
Not sure we want to pass the entire event to the saveConfig -- probably are passing data as well, no?